platform/security/platform-security-core-api/src/test/java/ddf/security/samlp/MetadataConfigurationParserTest.java

```diff
@@ -60,6 +60,7 @@
 
   @Before
   public void before() throws Exception {
+    System.setProperty("org.ops4j.pax.logging.DefaultServiceLog.level", "INFO");
```

Comment:
This is to avoid a Windows surefire issue where too many logs are being printed.

Reply:
❓ Should we conditionally do this only for Windows?

Reply:
See previous comment